src/services/subject.js

```
@@ -0,0 +1,38 @@
+const Subject = require('~/models/subject');
```

ESLint

[error] Extra semicolon.

(semi)